README.md

```diff
@@ -1,26 +1,27 @@
 # Weaviate Java client  <img alt='Weaviate logo' src='https://raw.githubusercontent.com/weaviate/weaviate/19de0956c69b66c5552447e84d016f4fe29d12c9/docs/assets/weaviate-logo.png' width='180' align='right' />
 
-A Java native client for weaviate.
+A Java native client for Weaviate.
 
 ## Usage
 
-In order to get start using the java client one needs to add it's dependency:
+To start using Weaviate Java client add this dependency to `pom.xml`:
 
 ```xml
+
 <dependency>
-  <groupId>io.weaviate</groupId>
-  <artifactId>client</artifactId>
-  <version>4.8.3</version>
+    <groupId>io.weaviate</groupId>
+    <artifactId>client</artifactId>
+    <version>4.8.3</version>
```

Comment:
We have a [prepare_release.sh](https://github.com/weaviate/java-client/blob/main/tools/prepare_release.sh#L26-L27) script that also updates the version here and it relies on the formatting of the README.md file. So either we bring back the casing to the state that it was before, or please adjust the prepare release script so that it would still work with your changes.

Reply:
I updated the script so that it would just re-paste any leading whitespace before the tag 👍